pkg/agent/manager/manager.go

```diff
@@ -235,31 +236,41 @@ func (m *manager) runBundleObserver(ctx context.Context) error {
 }
 
 func (m *manager) storeSVID(svidChain []*x509.Certificate) {
-	err := StoreSVID(m.svidCachePath, svidChain)
+	var err error
+	counter := agent_telemetry.StartManagerStoreSVIDCall(m.c.Metrics)
+	defer counter.Done(&err)
+	err = StoreSVID(m.svidCachePath, svidChain)
 	if err != nil {
 		m.c.Log.WithError(err).Warn("could not store SVID")
 	}
 }
 
```

Comment:
in these cases, the latency metric is not quite useful for anything IMO. What would we think about creating a new Telemetry package functionality for `Counters` (not `CallCounters`) that can indicate success status, much in the way as `CallCounter`~Done does today? (https://github.com/spiffe/spire/blob/master/pkg/common/telemetry/call.go#L66)